synapse/lib/cell.py

```diff
@@ -2232,12 +2232,12 @@ def getproc():
             if remove:
                 self.backupstreaming = False
 
-    async def isUserAllowed(self, iden, perm, gateiden=None):
-        user = self.auth.user(iden)
+    async def isUserAllowed(self, iden, perm, gateiden=None, default=False):
+        user = self.auth.user(iden)  # type: s_hiveauth.HiveUser
         if user is None:
             return False
 
-        return user.allowed(perm, gateiden=gateiden)
+        return user.allowed(perm, default=default, gateiden=gateiden)
```

Comment:
nit: kwarg ordering inconsistency with the above. perhaps move default to last?

Reply:
This was keeping the order for the callsig on HiveUser.allowed:

`def allowed(self, perm, default=None, gateiden=None):`

I was a bit hesitant to make the change to the `CellApi` callsig to insert the argument prior to `gateiden`; if someone had used that without a keyword arg they could break.

Reply:
bigger conversation probably: To help mitigate this issue in the future, should we consider using positional-only or keyword-only arguments in the CellApis? https://docs.python.org/3/glossary.html#term-parameter

Reply:
per discussion - lets let this as-is & address keyword only args in the future.